.github/workflows/release.yml

```diff
@@ -0,0 +1,322 @@
+          echo "SHOULD_RELEASE_BINARY=true" >> $GITHUB_ENV
+        fi
+
+  crate:
+    runs-on: ubuntu-latest
+    needs: check
+    if: needs.check.outputs.should_publish_crate == 'true'
+    env:
+      PROJECT_NAME: ${{ needs.check.outputs.project_name }}
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Install rust
+      uses: dtolnay/rust-toolchain@stable
+
+    - uses: Swatinem/rust-cache@v2
+
+    - uses: actions-rs/cargo@v1
+      with:
+        command: publish
+        args: --package ${{ env.PROJECT_NAME }}
+      env:
+        CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+
+  binary:
+    runs-on: ${{ matrix.os }}
+    needs: check
+    if: needs.check.outputs.should_release_binary == 'true'
+    env:
+      PROJECT_NAME: ${{ needs.check.outputs.project_name }}
+      PROJECT_VERSION: ${{ needs.check.outputs.project_version }}
+    strategy:
+      fail-fast: false
+      matrix:
+        name: [
+          # aarch64
+          linux-aarch64-gnu, linux-aarch64-musl, macos-aarch64, win64-aarch64-msvc,
+          # ? # arm / i686
+          # ? linux-arm-gnu, linux-arm-musl, linux-i686-gnu, linux-i686-musl, win32-msvc,
+          # x86_64
+          linux-x86_64-gnu, linux-x86_64-musl, macos-x86_64, win64-x86_64-gnu, win64-x86_64-msvc
+        ]
+        include:
+        - { name: 'linux-aarch64-gnu' , target: aarch64-unknown-linux-gnu   , os: ubuntu-latest , use-cross: true }
+        - { name: 'linux-aarch64-musl', target: aarch64-unknown-linux-musl  , os: ubuntu-latest , use-cross: true }
+        - { name: 'macos-aarch64'     , target: aarch64-apple-darwin        , os: macos-latest  ,                 }
+        - { name: 'win64-aarch64-msvc', target: aarch64-pc-windows-msvc     , os: windows-latest,                 }
+    # ? - { name: 'linux-arm-gnu'     , target: arm-unknown-linux-gnueabihf , os: ubuntu-latest , use-cross: true }
+    # ? - { name: 'linux-arm-musl'    , target: arm-unknown-linux-musleabihf, os: ubuntu-latest , use-cross: true }
+    # ? - { name: 'linux-i686-gnu'    , target: i686-unknown-linux-gnu      , os: ubuntu-latest , use-cross: true }
+    # ? - { name: 'linux-i686-musl'   , target: i686-unknown-linux-musl     , os: ubuntu-latest , use-cross: true }
+    # ? - { name: 'win32-msvc'        , target: i686-pc-windows-msvc        , os: windows-latest,                 }
+        - { name: 'linux-x86_64-gnu'  , target: x86_64-unknown-linux-gnu    , os: ubuntu-latest ,                 }
+        - { name: 'linux-x86_64-musl' , target: x86_64-unknown-linux-musl   , os: ubuntu-latest , use-cross: true }
+        - { name: 'macos-x86_64'      , target: x86_64-apple-darwin         , os: macos-latest  ,                 }
+        - { name: 'win64-x86_64-gnu'  , target: x86_64-pc-windows-gnu       , os: windows-latest,                 }
+        - { name: 'win64-x86_64-msvc' , target: x86_64-pc-windows-msvc      , os: windows-latest,                 }
```

Comment:
This leads to https://github.com/near/cargo-near/issues/4, I haven't tested `cargo-near` builds on Windows lately.